drivers/include/net/netdev.h

```diff
@@ -54,6 +54,7 @@ extern "C" {
 #include <sys/uio.h>
 
 #include "net/netstats.h"
+#include "net/l2filter.h"
```

Comment:
yeah should be fine as long there is no global or external variable definition that could result in _undefined reference_ or _unused variable_ error when the module is not used.